### ISSUE TITLE
Add web clipper just like onenote or Evernote？

### ISSUE THREAD

**Issue by Bugatti100Peagle** [nextcloud/notes#403]
**I want to clip full web page not url or txt**
When I want to clip a web page, I click on "share" button and note will clip the full page for me. When I open it next, it's like it saw.

**Describe alternatives you've considered**
[Evernote Clipper](https://www.yinxiang.com/webclipper/)
[Onenote Web Clipper](https://www.onenote.com/clipper)
[WizNote Clipper](https://www.wiz.cn/zh-cn/downloads-webclipper.html)

**Comment by korelstar** [comment]
This could be a nice feature, but currently, it is out of scope for this app.

**Comment by github-account1111** [comment]
I think a better option might be adding wallabag / archival support to the bookmarks app. This will take care of web-clipping AND link rot.

**Comment by Boki4d** [comment]
I think notes will improve after time and gain a higher adoption rate, especially with the new text editor (i'm hoping for image support). Therefore it would make sense to think about a nextcloud-wide webclipper that would either save to bookmarks or notes.

**Comment by GambaJo** [comment]
This would be great.

**Comment by stefan-niedermann** [comment]
@GambaJo no offense, honestly :slightly_smiling_face: 

### How to use GitHub ([source](https://github.com/nextcloud/server/blob/98a2e3b628f29260b22b3ce98f553404685eadf0/.github/ISSUE_TEMPLATE/Feature_request.md#how-to-use-github))

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by adx442** [comment]
I see a path to integrate this functionality cleanly with Notes and Nextcloud:  [SingleFile](https://github.com/gildas-lormeau/SingleFile) is on Github and OSS, does a much better job of capture than Wallabag, and stores the entirety of a page in one HTML file.  Those captured pages could be stored in a Notes subdirectory with a relative link placed in the Note, for minimum overhead/refactoring.  Best of both worlds?

**Comment by janxkoci** [comment]
FYI there is also a "web clipper" for [Google Drive](https://chrome.google.com/webstore/detail/save-to-google-drive/gmbmikajjgmnabiglmofipeabaddhgne). It's not as usefull as e.g. Evernote, but still, can inspire some UX/features ideas..

**Comment by timkgh** [comment]
+1 for a web clipper. I haven't used [Nextcloud Bookmarks](https://apps.nextcloud.com/apps/bookmarks), but it claims that it can save full pages. This could be another potential integration.

**Comment by Spartachetto** [comment]
Maybe @marcelklehr of Nextcloud Bookmarks could contribute to the discussion with some useful feedback...

**Comment by marcelklehr** [comment]
> I haven't used Nextcloud Bookmarks, but it seems it can save full pages.

Yep, Nextcloud Bookmarks archives bookmarked pages in full-text, allowing search and clutter-free read-it-later. I'm not sure how this could be integrated into the Notes app, though.

**Comment by wokawoka** [comment]
+1
a web clipper is really needed

**Comment by juliushaertl** [comment]
@wokawoka Please use github reactions instead of +1 comments

**Comment by juliushaertl** [comment]
We could actually use link preview widgets for this. Now we don't want all links to be rendered as a embed but I could imagine making use of the bookmarks app parts then and have a link preview for bookmarks app internal links that inlines the bookmarked page somehow
